src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+
+    public interface Printable{
```

Comment:
Comments needed here for the interface.

Reply:
Ouyang, do u have email? I have problems regarding running addressbook 4.

Reply:
@viviantan95 My email is a0127481@u.nus.edu. You can also contact me via Slack. What issues have you encountered?I am currently in Malaysia though. I will take a look on Thursday if it is not too late. You can also ask in the public channel on Slack or from you teammates before that.